catchpoint-mixin/README.md

```diff
@@ -0,0 +1,94 @@
+
+- Catchpoint overview
+- Catchpoint web performance by test name
+- Catchpoint web performance by node name
+
+and the following alerts:
+
+- CatchpointHighDNSResolutionTime
+- CatchpointTotalTimeExceeded
+- CatchpointContentLoadingDelays
+- CatchpointHighServerResponseTime
+- CatchpointHighFailedRequestRatio
+
+## Catchpoint overview
+
+The Catchpoint overview dashboard provides details on alerts, load times, request success rates, connections, and errors.
+
+![Catchpoint overview dashboard]()
+
+## Catchpoint web performance by test name
+
+The Catchpoint web performance by test name dashboard provides details on page completion times, response sizes, network connections, requests, content types, and errors. These metrics are all grouped by test name.
+
+![Catchpoint web performance by test name]()
+
+## Catchpoint web performance by node name
+
+The Catchpoint web performance by node name dashboard provides details on page completion times, response sizes, network connections, requests, content types, and errors. These metrics are all grouped by node name.
+
+![Catchpoint web performance by node name]()
+
+## Alerts Overview
+
+| Alert                            | Summary                                                                                                                                                       |
+| -------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| CatchpointHighDNSResolutionTime  | DNS resolution time is high, which could indicate DNS server issues or misconfigurations affecting website accessibility.                                     |
+| CatchpointTotalTimeExceeded      | Webpage takes too long to load, potentially indicating issues with server response, network latency, or resource-heavy pages.                                 |
+| CatchpointContentLoadingDelays   | Significant delays in content loading could impact user experience and webpage usability.                                                                     |
+| CatchpointHighServerResponseTime | High server response times can lead to slow user experiences and decreased satisfaction.                                                                      |
+| CatchpointHighFailedRequestRatio | High ratio of failed requests to total requests could indicate deteriorations in service quality, which could affect user experience and compliance with SLA. |
+
+Default thresholds can be configured in `config.libsonnet`
+
+```js
+{
+  _config+:: {
+  alertsHighServerResponseTime: 1000, //ms
+  alertsHighServerResponseTimePercent: 1.2, // 120%
+  alertsTotalTimeExceeded: 5000, //ms
+  alertsTotalTimeExceededPercent: 1.2, // 120%
+  alertsHighDNSResolutionTime: 500, //ms
+  alertsHighDNSResolutionTimePercent: 1.2, // 120%
+  alertsContentLoadingDelay: 3000, //ms
```

Comment:
a 3 second delay seems surprisingly long, is that really what we want to recommend?

Reply:
Yea i think this is a bit long, It was on the spec, but I personally think pulling it down to 1500 ms would be better. What do you think @Dasomeone?

Reply:
Yes, I think that makes sense! Let's go with that and revisit in the future if a longer period seems useful